pikaur/config.py

```diff
@@ -174,6 +174,10 @@ def get_config_path() -> str:
         },
     },
     'misc': {
+        'UseDoas': {
```

Comment:
or better

```python
'PrivilegeEscalationTool': {
  'type': 'str',
  'default': 'sudo',
},
```

Reply:
Yes, this is an option, but a potential problem here is that not all such tools would behave the same with the rest of the program. For example, `su` is a Privilege Escalation Tool, but it wouldn't work with the rest of the program. Ultimately it's up to you.

Reply:
in the docs you could mention what at the moment only `sudo` and `doas` are supported + mention about `permit persist`

Reply:
Implemented